build-data/generated/tracking_parameters.json

```diff
@@ -0,0 +1,902 @@
+{
+    "totalSites": 144555,
+    "params": {
+        "gclid": {
+            "prevalence": 0.596,
+            "exampleSites": [
+                "bearswire.usatoday.com",
+                "mobile.southwest.com",
+                "www.cplusplus.com",
+                "jp.spankbang.com",
+                "au.pcmag.com/web-hosting-services/8578/the-best-vps-web-hosting-services-for-2020",
+                "www.merriam-webster.com",
+                "puzzles.msn.com",
+                "vimeo.com",
+                "community.babycenter.com/post/a7884385/straining_crying_during_bowel_movements_for_breastfed_newborn",
+                "www.travelmath.com/time-change/from/Spain/to/Argentina"
+            ],
+            "requests3p": {
+                "prevalence": 0.285,
+                "entities": {
+                    "Google LLC": 0.743,
```

Comment:
I start to wonder if domains wouldn't be better here - e.g. I think it'd be nice to know if it's google-analytics or google ads or GTM or firebase collecting those IDs I think?

Reply:
We can do both

Reply:
this is great, thank you!